### PR TITLE
Product variants and assets

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | 
| Versions      | 

To add as follow-up: the function of "Set as main photo", when finalized.

### CODE REVIEW

File: docs/shop_administration/product_variants_and_assets.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+You can generate variants automatically based on the configured attributes.
+
+To generate variants:
+
+1. Go to the **Variants** tab in product view and click **Generate variants**.
+
+![Generating product variants](img/product_variants_generate.png)
+
+1. For each attribute listed, select the values you want to create variants for.
+1. Confirm with **Generate variants**.
```
Comment (adamwojs) [inline docs/shop_administration/product_variants_and_assets.md:35]:
```suggestion
1. Go to the **Variants** tab in product view and click **Generate variants**.

![Generating product variants](img/product_variants_generate.png)

2. For each attribute listed, select the values you want to create variants for.
3. Confirm with **Generate variants**.
```

Comment (DominikaK) [inline docs/shop_administration/product_variants_and_assets.md:35]:
```suggestion
1. Go to the **Variants** tab in product view and click **Generate variants**.

![Generating product variants](img/product_variants_generate.png)

2\. For each attribute listed, select the values you want to create variants for.

3\. Confirm with **Generate variants**.
```

File: docs/shop_administration/product_variants_and_assets.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+Product variants enable you to have multiple versions of one product, differing in some characteristics.
+
+Typical examples are a t-shirt in different colors, or a laptop with different hard disk sizes.
+
+You can create variants based on product attributes.
+To use an attribute to create variants, mark the attribute as **Used for product variants** when creating a product type.
+
+![Product type definition with attributes used for variants](img/product_type_attr_used_for_variants.png)
+
+If a product has one attribute that can be used for variants, you can create variants for this attribute.
```
Comment (MagdalenaZuba) [inline docs/shop_administration/product_variants_and_assets.md:18]:
```suggestion
```
already explained above, so I would skip repetition

File: docs/shop_administration/product_variants_and_assets.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+1. Go to the **Variants** tab in product view and click **Generate variants**.
+
+![Generating product variants](img/product_variants_generate.png)
+
+2\. For each attribute listed, select the values you want to create variants for.
+
+3\. Confirm with **Generate variants**.
+
+You can now see a list of all variants for the possible combinations of selected attribute values.
```
Comment (MagdalenaZuba) [inline docs/shop_administration/product_variants_and_assets.md:38]:
```suggestion
You can now see a list of all possible variants for the selected attribute values.
```

File: docs/shop_administration/product_variants_and_assets.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+![Product variant list with option ot delete a variant](img/product_variants_delete.png)
+
+### Manually creating variants
+
+If needed, you can manually create any missing variants, including ones you previously deleted.
+To do this, click **Add** next to the variant list, enter a product code and select attribute values for the new variant.
+
+### Managing variants
+
+You set up product availability separately for each product variant.
```
Comment (MagdalenaZuba) [inline docs/shop_administration/product_variants_and_assets.md:54]:
```suggestion
You can set up product availability separately for each product variant.
```

File: docs/shop_administration/product_variants_and_assets.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+You can also set up prices per product variant.
+The prices will override prices set for the main product.
+If you do not define a price for a variant, the price of the main product is used.
+
+## Product assets
+
+You can add assets to products in the form of images.
+Assets can be assigned to the base product, and to one or more of its variants.
+
+To add assets to a product, in the product's **Assets** tab browse or drag and drop to select image files from disk,
```
Comment (MagdalenaZuba) [inline docs/shop_administration/product_variants_and_assets.md:65]:
```suggestion
To add assets to a product, in the product's **Assets** tab browse or drag and drop image files from disk,
```

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

File: docs/shop_administration/manage_products.md [hunk 6]
```diff
@@ -39,9 +39,19 @@ When adding attributes to product types, you can add both individual attributes,
...
+For every attribute, you can select **Used for product variants**.
+Attributes that have this option are used to [automatically generate product variants](product_variants_and_assets.md#generating-variants).
+
 ## Product availability and stock
 
-To control a product's availability and the available stock, use the product's **Availability** tab.
+To control a product's availability and the available stock, use the product's or product variant's **Availability** tab.
+
+If a product can have variants, you must set availability for each variant separately.
+If a product cannot have variants, you can set availability for the whole product.
```
Comment (DominikaK) [inline docs/shop_administration/manage_products.md:54]:
```suggestion
To control a product's availability and the available stock, use the product's **Availability** tab.
If a product can have variants, you must set availability for each variant separately.
```
In this case, is it totally clear that without variants availability is set for base products? We had a problem with clarity in the corresponding dev-doc PR.

File: docs/shop_administration/product_variants_and_assets.md (hunk 2 above)
Comment (DominikaK) [reply docs/shop_administration/product_variants_and_assets.md:18]: I wanted to contrast here: one attribute -> plain variants, multiple attributes: variants for all combinations. Maybe clarify to "only one attribute"?

File: docs/shop_administration/manage_products.md (hunk 6 above)
Comment (MagdalenaZuba) [reply docs/shop_administration/manage_products.md:54]: For me it is more obvious like that, but if we had problems with it, I'm ok with 'don't wash your baby in the washing machine' type of instruction :)

**Review by MagdalenaZuba — APPROVED** [review]

**Pull request merged**